From d1eb7bba54e75caf0ddb15bc611a7c1eca8296f2 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Wed, 5 Mar 2025 08:01:59 +0800
Subject: [PATCH] libxml2: reset to working commit

---
 packages/libxml2.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/libxml2.cmake b/packages/libxml2.cmake
index 2148bd5..751692f 100644
--- a/packages/libxml2.cmake
+++ b/packages/libxml2.cmake
@@ -6,6 +6,7 @@ ExternalProject_Add(libxml2
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--sparse --filter=tree:0"
     GIT_CLONE_POST_COMMAND "sparse-checkout set --no-cone /* !result !test"
+    GIT_RESET 66fdf94c5518547c12311db1e4dc0485acf2a2f8
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} CONF=1 cmake -H<SOURCE_DIR> -B<BINARY_DIR>
         -G Ninja
-- 
2.48.1.windows.1

