From 1b42b848ba5b97154770d92548d54ff6050b2c51 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Mon, 20 Nov 2023 20:33:39 +0800
Subject: [PATCH] rustup: fix cargo-c install

---
 toolchain/rustup.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/toolchain/rustup.cmake b/toolchain/rustup.cmake
index 5d199d1..51cc153 100644
--- a/toolchain/rustup.cmake
+++ b/toolchain/rustup.cmake
@@ -6,7 +6,7 @@ ExternalProject_Add(rustup
         curl -sSf https://sh.rustup.rs |
         sh -s -- -y --default-toolchain stable --target x86_64-pc-windows-gnu,i686-pc-windows-gnu --no-modify-path --profile minimal
     BUILD_COMMAND ${EXEC} rustup update
-    INSTALL_COMMAND ${EXEC} cargo install cargo-c --profile=release-strip --features=vendored-openssl
+    INSTALL_COMMAND ${EXEC} cargo install cargo-c --profile=release-strip --features=vendored-openssl --locked
     LOG_CONFIGURE 1 LOG_BUILD 1 LOG_INSTALL 1
 )
 
-- 
2.42.0

