From 48bd1f985a1bf89c7c9cfb57a5d13ec8ff45a0c6 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Wed, 13 Mar 2024 21:07:15 +0800
Subject: [PATCH] ffmpeg: reset to working commit

---
 packages/ffmpeg.cmake | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packages/ffmpeg.cmake b/packages/ffmpeg.cmake
index e620f2f..30d65a1 100644
--- a/packages/ffmpeg.cmake
+++ b/packages/ffmpeg.cmake
@@ -49,6 +49,9 @@ ExternalProject_Add(ffmpeg
     GIT_REPOSITORY https://github.com/FFmpeg/FFmpeg.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
+    GIT_REMOTE_NAME origin
+    GIT_TAG master
+    GIT_RESET daca5d1241f33c9a457be1a9afcf058206c263b3
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} CONF=1 <SOURCE_DIR>/configure
         --cross-prefix=${TARGET_ARCH}-
-- 
2.43.0.windows.1

