From 36e028092b97813a26daddf91c39b877324852b2 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Wed, 5 Aug 2026 20:53:35 +0800
Subject: [PATCH] ffmpeg: reset to working commit

---
 packages/ffmpeg.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/packages/ffmpeg.cmake b/packages/ffmpeg.cmake
index b9ab7a3..3cd67ef 100644
--- a/packages/ffmpeg.cmake
+++ b/packages/ffmpeg.cmake
@@ -48,6 +48,8 @@ ExternalProject_Add(ffmpeg
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--sparse --filter=tree:0"
     GIT_CLONE_POST_COMMAND "sparse-checkout set --no-cone /* !tests/ref/fate"
+    GIT_REMOTE_NAME origin
+    GIT_RESET 9862dd83b1645b0c6e39b318aefb896138ffb08e
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} CONF=1 <SOURCE_DIR>/configure
         --cross-prefix=${TARGET_ARCH}-
-- 
2.45.2.windows.1

