From f8e99bb55853f2b2f67c755b650a7ffc95f6cacf Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Fri, 11 Oct 2024 22:10:17 +0800
Subject: [PATCH] vulkan-header: reset to 1.3.297

---
 packages/vulkan-header.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/vulkan-header.cmake b/packages/vulkan-header.cmake
index fcd9e829e..1b6eb9ffd 100644
--- a/packages/vulkan-header.cmake
+++ b/packages/vulkan-header.cmake
@@ -5,6 +5,7 @@ ExternalProject_Add(vulkan-header
     UPDATE_COMMAND ""
     GIT_REMOTE_NAME origin
     GIT_TAG main
+    GIT_RESET 14345dab231912ee9601136e96ca67a6e1f632e7
     CONFIGURE_COMMAND ${EXEC} CONF=1 cmake -H<SOURCE_DIR> -B<BINARY_DIR>
         -G Ninja
         -DCMAKE_BUILD_TYPE=Release
