From 2919bd602d21ed5dba7dbba9f9147871acbb02bf Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Mon, 25 Sep 2023 19:49:06 +0800
Subject: [PATCH] openal-soft: clamp to working commit

---
 packages/openal-soft.cmake | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packages/openal-soft.cmake b/packages/openal-soft.cmake
index 992d916..d4f452a 100644
--- a/packages/openal-soft.cmake
+++ b/packages/openal-soft.cmake
@@ -3,6 +3,9 @@ ExternalProject_Add(openal-soft
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
     UPDATE_COMMAND ""
+    GIT_REMOTE_NAME origin
+    GIT_TAG master
+    GIT_RESET 2e24d1ed773c2fb0f5b85acb9c02acb9c8b5462d
     CONFIGURE_COMMAND ${EXEC} cmake -H<SOURCE_DIR> -B<BINARY_DIR>
         -DCMAKE_BUILD_TYPE=Release
         -DCMAKE_INSTALL_PREFIX=${MINGW_INSTALL_PREFIX}
-- 
2.42.0

