From 63157a7a91c855cbb1d48e78e41fed105fb80883 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Mon, 27 Nov 2023 18:50:57 +0800
Subject: [PATCH] vulkan: reset to working commit

---
 packages/vulkan-header.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/vulkan-header.cmake b/packages/vulkan-header.cmake
index bf18b36..5351cf5 100644
--- a/packages/vulkan-header.cmake
+++ b/packages/vulkan-header.cmake
@@ -5,6 +5,7 @@ ExternalProject_Add(vulkan-header
     UPDATE_COMMAND ""
     GIT_REMOTE_NAME origin
     GIT_TAG main
+    GIT_RESET 7d92fe7b687eb92dafb5d344d1f4c817780cd3ce
     CONFIGURE_COMMAND ${EXEC} CONF=1 cmake -H<SOURCE_DIR> -B<BINARY_DIR>
         -G Ninja
         -DCMAKE_BUILD_TYPE=Release
-- 
2.42.0

