From 3590d048911d7376b45c45cf7743ad87b5fae09a Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Thu, 28 Sep 2023 20:38:56 +0800
Subject: [PATCH] libjxl: clamp to working commit

---
 packages/libjxl.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/libjxl.cmake b/packages/libjxl.cmake
index ef38040..3c353cb 100644
--- a/packages/libjxl.cmake
+++ b/packages/libjxl.cmake
@@ -14,6 +14,7 @@ ExternalProject_Add(libjxl
     GIT_CLONE_FLAGS "--filter=tree:0"
     GIT_REMOTE_NAME origin
     GIT_TAG main
+    GIT_RESET d895ec88f42c1a0031d0e419fcc53d15e2fd92fb
     GIT_SUBMODULES ""
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ""
-- 
2.41.0

