From b34d7ec2b449bc932558cfc8d8c2178f2bd8bfcc Mon Sep 17 00:00:00 2001
From: nanachisol <nanachisol@github.com>
Date: Sat, 19 Apr 2025 17:24:20 +0800
Subject: [PATCH] add Andarwinux librempeg patch

---
 packages/ffmpeg.cmake | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/packages/ffmpeg.cmake b/packages/ffmpeg.cmake
index 0234a11..ae79208 100644
--- a/packages/ffmpeg.cmake
+++ b/packages/ffmpeg.cmake
@@ -47,7 +47,7 @@ ExternalProject_Add(ffmpeg
         frei0r
         vvenc
         codec2
-    GIT_REPOSITORY https://github.com/Andarwinux/FFmpeg.git
+    GIT_REPOSITORY https://github.com/librempeg/librempeg.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--depth=1 --sparse --filter=tree:0"
     GIT_PROGRESS TRUE
@@ -65,7 +65,6 @@ ExternalProject_Add(ffmpeg
         --enable-gpl
         --enable-version3
         --enable-nonfree
-        --enable-postproc
         --enable-avisynth
         --enable-vapoursynth
         --enable-libass
-- 
2.49.0.windows.1

