From b9521a02de972d0382594ecb784a9aeee79c908f Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Mon, 22 Jan 2024 20:58:29 +0800
Subject: [PATCH] libjxl: reset to working commit

---
 packages/libjxl.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/libjxl.cmake b/packages/libjxl.cmake
index cbc9ea3..f6e413b 100644
--- a/packages/libjxl.cmake
+++ b/packages/libjxl.cmake
@@ -12,6 +12,7 @@ ExternalProject_Add(libjxl
     GIT_CLONE_FLAGS "--filter=tree:0"
     GIT_REMOTE_NAME origin
     GIT_TAG main
+    GIT_RESET 3671f93c9c611ec7d6bf96a5cf4749b1480c5c27
     GIT_SUBMODULES ""
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ""
-- 
2.43.0.windows.1

