From e8d4fc3222084a1c59f8a5517b646c40cdedcb6a Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Fri, 14 Jun 2024 08:48:55 +0800
Subject: [PATCH] vapoursynth: fix typo in c81bc71

---
 packages/vapoursynth.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/packages/vapoursynth.cmake b/packages/vapoursynth.cmake
index 00b2a57a..c34b8a96 100644
--- a/packages/vapoursynth.cmake
+++ b/packages/vapoursynth.cmake
@@ -69,9 +69,9 @@ ExternalProject_Add_Step(vapoursynth manual-install
     WORKING_DIRECTORY <SOURCE_DIR>
     # Copying header
     COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/VapourSynth.h ${MINGW_INSTALL_PREFIX}/include/vapoursynth/VapourSynth.h
-    COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/VapourSynth.h ${MINGW_INSTALL_PREFIX}/include/vapoursynth/VapourSynth4.h
+    COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/VapourSynth4.h ${MINGW_INSTALL_PREFIX}/include/vapoursynth/VapourSynth4.h
     COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/VSScript.h ${MINGW_INSTALL_PREFIX}/include/vapoursynth/VSScript.h
-    COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/VSScript.h ${MINGW_INSTALL_PREFIX}/include/vapoursynth/VSScript4.h
+    COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/VSScript4.h ${MINGW_INSTALL_PREFIX}/include/vapoursynth/VSScript4.h
     # Copying libs
     ${vapoursynth_manual_install_copy_lib}
     # Copying .pc files
-- 
2.45.1.windows.1

