From d334f08bbe1106e5b177e8a5f0a6589ba9ddcf6b Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Wed, 25 Jun 2025 18:20:23 +0800
Subject: [PATCH] spirv-headers: revert to working commit

---
 packages/spirv-headers.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/spirv-headers.cmake b/packages/spirv-headers.cmake
index 567c2b1..24fdf9a 100644
--- a/packages/spirv-headers.cmake
+++ b/packages/spirv-headers.cmake
@@ -4,6 +4,7 @@ ExternalProject_Add(spirv-headers
     GIT_CLONE_FLAGS "--filter=tree:0"
     GIT_REMOTE_NAME origin
     GIT_TAG main
+    GIT_RESET 2a611a970fdbc41ac2e3e328802aed9985352dca
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ""
     BUILD_COMMAND ""
-- 
2.48.1.windows.1

