From dd32c66729950933ae6b46ef8a0c2b2cba32fcbc Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Tue, 31 Dec 2024 18:59:53 +0800
Subject: [PATCH] libass: disable unneeded features

---
 packages/libass.cmake | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/packages/libass.cmake b/packages/libass.cmake
index d6aba079d..ed27477ae 100644
--- a/packages/libass.cmake
+++ b/packages/libass.cmake
@@ -20,6 +20,11 @@ ExternalProject_Add(libass
         -Ddirectwrite=enabled
         -Dasm=enabled
         -Dlibunibreak=enabled
+        -Dtest=disabled
+        -Dcompare=disabled
+        -Dprofile=disabled
+        -Dfuzz=disabled
+        -Dcheckasm=disabled
     BUILD_COMMAND ${EXEC} ninja -C <BINARY_DIR>
     INSTALL_COMMAND ${EXEC} ninja -C <BINARY_DIR> install
     LOG_DOWNLOAD 1 LOG_UPDATE 1 LOG_CONFIGURE 1 LOG_BUILD 1 LOG_INSTALL 1
-- 
2.47.1.windows.1

