From 942031f3e82f32bb795ea1d9fa0c148dd4cee983 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Thu, 22 Sep 2022 13:27:50 +0800
Subject: [PATCH 1/2] rubberband: clamp to working commit

---
 packages/rubberband.cmake | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/packages/rubberband.cmake b/packages/rubberband.cmake
index 22be92e..9755ff1 100644
--- a/packages/rubberband.cmake
+++ b/packages/rubberband.cmake
@@ -1,9 +1,7 @@
 ExternalProject_Add(rubberband
     GIT_REPOSITORY https://github.com/breakfastquay/rubberband.git
     SOURCE_DIR ${SOURCE_LOCATION}
-    GIT_SHALLOW 1
-    GIT_REMOTE_NAME origin
-    GIT_TAG default
+    GIT_TAG 2eb9a200525461f30fe3d21f399d91f525a09ba2
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} meson <BINARY_DIR> <SOURCE_DIR>
         --prefix=${MINGW_INSTALL_PREFIX}
-- 
2.37.1.windows.1

