From 46cc241cb7891edb15d0ff25205b2d657b3df6b6 Mon Sep 17 00:00:00 2001
From: Andarwinux <Andarwinux@users.noreply.github.com>
Date: Tue, 23 Apr 2024 00:00:00 +0000
Subject: [PATCH] ffmpeg: fix build

---
 cmake/packages_check.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/cmake/packages_check.cmake b/cmake/packages_check.cmake
index 6f55c4660..d0985529c 100644
--- a/cmake/packages_check.cmake
+++ b/cmake/packages_check.cmake
@@ -17,7 +17,8 @@ elseif(COMPILER_TOOLCHAIN STREQUAL "clang")
     set(vapoursynth_manual_install_copy_lib COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/VSScript.lib ${MINGW_INSTALL_PREFIX}/lib/VSScript.lib
                                             COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/VapourSynth.lib ${MINGW_INSTALL_PREFIX}/lib/VapourSynth.lib)
     set(ffmpeg_extra_libs "-lc++")
-    set(ffmpeg_hardcoded_tables "--enable-hardcoded-tables")
+    #set(ffmpeg_hardcoded_tables "--enable-hardcoded-tables")
+    #broken by recent aac commits
     set(mpv_lto_mode "-Db_lto_mode=thin")
     set(mpv_copy_debug COMMAND ${CMAKE_COMMAND} -E copy <BINARY_DIR>/mpv.pdb ${CMAKE_CURRENT_BINARY_DIR}/mpv-debug/mpv.pdb)
     if(CLANG_PACKAGES_LTO)
