From d8ace2851acfb5e7f37e4a39c0f58fd1d4022537 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Wed, 23 Jul 2025 22:20:40 +0800
Subject: [PATCH] harfbuzz: reset to working commit

---
 packages/harfbuzz.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/harfbuzz.cmake b/packages/harfbuzz.cmake
index 5248f25..f2eb305 100644
--- a/packages/harfbuzz.cmake
+++ b/packages/harfbuzz.cmake
@@ -6,6 +6,7 @@ ExternalProject_Add(harfbuzz
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_REMOTE_NAME origin
     GIT_TAG main
+    GIT_RESET c5047aa86e491dcc571e8ca3ea1ea0bd36469752
     GIT_CLONE_FLAGS "--sparse --filter=tree:0"
     GIT_CLONE_POST_COMMAND "sparse-checkout set --no-cone /* !test"
     UPDATE_COMMAND ""
-- 
2.48.1.windows.1

