From 1e6786825a3ef3594859d306cf36a825540039c8 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Sat, 18 Feb 2023 08:50:15 +0000
Subject: [PATCH] vulkan-header: rollback to fix libplacebo build

---
 packages/vulkan-header.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/vulkan-header.cmake b/packages/vulkan-header.cmake
index c248fc2..b728bb4 100644
--- a/packages/vulkan-header.cmake
+++ b/packages/vulkan-header.cmake
@@ -4,7 +4,7 @@ ExternalProject_Add(vulkan-header
     GIT_CLONE_FLAGS "--filter=tree:0"
     UPDATE_COMMAND ""
     GIT_REMOTE_NAME origin
-    GIT_TAG main
+    GIT_TAG v1.3.240
     CMAKE_ARGS
         -DCMAKE_BUILD_TYPE=Release
         -DCMAKE_INSTALL_PREFIX=${MINGW_INSTALL_PREFIX}
-- 
2.39.2
