From ecc8cc0d48155560e90c64759bd8f4c90563d914 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Mon, 11 Nov 2024 10:01:37 +0800
Subject: [PATCH] nvcodec-headers: change to mirror

---
 packages/nvcodec-headers.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/nvcodec-headers.cmake b/packages/nvcodec-headers.cmake
index 77fa0988b..baebfbad2 100644
--- a/packages/nvcodec-headers.cmake
+++ b/packages/nvcodec-headers.cmake
@@ -1,5 +1,5 @@
 ExternalProject_Add(nvcodec-headers
-    GIT_REPOSITORY https://git.videolan.org/git/ffmpeg/nv-codec-headers.git
+    GIT_REPOSITORY https://github.com/zhongflyTeam/nv-codec-headers.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
     UPDATE_COMMAND ""

