From 3e1c687efedae67578bbc913b0f0dac1e037b0b6 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Mon, 10 Feb 2025 20:12:59 +0800
Subject: [PATCH] ffmpeg: revert to working commit

---
 packages/ffmpeg.cmake | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packages/ffmpeg.cmake b/packages/ffmpeg.cmake
index 49a0f6e95..53f7f121e 100644
--- a/packages/ffmpeg.cmake
+++ b/packages/ffmpeg.cmake
@@ -50,6 +50,9 @@ ExternalProject_Add(ffmpeg
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--sparse --filter=tree:0"
     GIT_CLONE_POST_COMMAND "sparse-checkout set --no-cone /* !tests/ref/fate"
+    GIT_REMOTE_NAME origin
+    GIT_TAG master
+    GIT_RESET e838e6cbe2d8ab4ad9cea85e04e545ff27e97bdb
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} CONF=1 <SOURCE_DIR>/configure
         --cross-prefix=${TARGET_ARCH}-
-- 
2.47.1.windows.1

