From 99a21947077cefffdbc1ef9d39bb8863a16e0f88 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Fri, 25 Nov 2022 12:36:06 +0000
Subject: [PATCH] spirv-tools: fix build

---
 packages/spirv-tools.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/spirv-tools.cmake b/packages/spirv-tools.cmake
index a3edf76..f94a324 100644
--- a/packages/spirv-tools.cmake
+++ b/packages/spirv-tools.cmake
@@ -3,6 +3,7 @@ ExternalProject_Add(spirv-tools
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_SHALLOW 1
     UPDATE_COMMAND ""
+    PATCH_COMMAND ${EXEC} curl -sL https://github.com/KhronosGroup/SPIRV-Tools/pull/5002.patch | git am --3way --whitespace=fix
     CONFIGURE_COMMAND ""
     BUILD_COMMAND ""
     INSTALL_COMMAND ""
-- 
2.38.1

