From 50bf5b802b220e7127c67800a5e3a839298ca2be Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Sun, 30 Jul 2023 21:13:32 +0800
Subject: [PATCH] brotli: clamp to working commit

---
 packages/brotli.cmake | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packages/brotli.cmake b/packages/brotli.cmake
index c2156e1..f94b927 100644
--- a/packages/brotli.cmake
+++ b/packages/brotli.cmake
@@ -4,6 +4,9 @@ ExternalProject_Add(brotli
     GIT_CLONE_FLAGS "--sparse --filter=tree:0"
     GIT_CLONE_POST_COMMAND "sparse-checkout set --no-cone /* !tests"
     UPDATE_COMMAND ""
+    GIT_REMOTE_NAME origin
+    GIT_TAG master
+    GIT_RESET 0300be36ba019c55d2edc48353270fa18008d49c
     CONFIGURE_COMMAND ${EXEC} cmake -H<SOURCE_DIR> -B<BINARY_DIR>
         -DCMAKE_INSTALL_PREFIX=${MINGW_INSTALL_PREFIX}
         -DCMAKE_TOOLCHAIN_FILE=${TOOLCHAIN_FILE}
-- 
2.41.0.windows.1

