From c6448511697a03157e73eda07d150a725a1a35bb Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Wed, 17 Apr 2024 02:35:58 +0800
Subject: [PATCH] shaderc: reset to working commit

---
 packages/shaderc.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/shaderc.cmake b/packages/shaderc.cmake
index 105344ec..4805ba51 100644
--- a/packages/shaderc.cmake
+++ b/packages/shaderc.cmake
@@ -7,6 +7,7 @@ ExternalProject_Add(shaderc
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_REMOTE_NAME origin
     GIT_TAG main
+    GIT_RESET e6187c7
     GIT_CLONE_FLAGS "--filter=tree:0"
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} LTO_JOB=1 CONF=1 cmake -H<SOURCE_DIR> -B<BINARY_DIR>
-- 
2.43.0.windows.1

