From 0e758c3b4446a36c017da8e51479e73e14cf6d93 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Wed, 3 Jan 2024 20:53:15 +0800
Subject: [PATCH] libzvbi: drop patch

---
 packages/libzvbi.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/packages/libzvbi.cmake b/packages/libzvbi.cmake
index 42f4827..2523fc9 100644
--- a/packages/libzvbi.cmake
+++ b/packages/libzvbi.cmake
@@ -7,7 +7,6 @@ ExternalProject_Add(libzvbi
     GIT_CLONE_FLAGS "--filter=tree:0"
     GIT_REMOTE_NAME origin
     GIT_TAG main
-    PATCH_COMMAND ${EXEC} git am --3way ${CMAKE_CURRENT_SOURCE_DIR}/libzvbi-*.patch
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} <SOURCE_DIR>/autogen.sh && CONF=1 <SOURCE_DIR>/configure
         --host=${TARGET_ARCH}
-- 
2.42.0

