From d6a63d0c99931e1949011fd2b9e23849a1267edf Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Thu, 24 Oct 2024 08:03:21 +0800
Subject: [PATCH] spirv-headers: reset to working commit

---
 packages/spirv-headers.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/spirv-headers.cmake b/packages/spirv-headers.cmake
index 567c2b1f5..4eb024325 100644
--- a/packages/spirv-headers.cmake
+++ b/packages/spirv-headers.cmake
@@ -4,6 +4,7 @@ ExternalProject_Add(spirv-headers
     GIT_CLONE_FLAGS "--filter=tree:0"
     GIT_REMOTE_NAME origin
     GIT_TAG main
+    GIT_RESET 252dc2df08f58e0e50c8437edc0e77eacdfb7559
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ""
     BUILD_COMMAND ""
-- 
2.47.0.windows.1

