From 52fc8a4c74406168ccdd702897f4dccdb88c9e6f Mon Sep 17 00:00:00 2001
From: Andarwinux <Andarwinux@users.noreply.github.com>
Date: Thu, 25 Jan 2024 00:00:00 +0000
Subject: [PATCH] openal-soft: fix ffmpeg configure

---
 packages/openal-soft.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/openal-soft.cmake b/packages/openal-soft.cmake
index c88ac7e48..e173bc4ce 100644
--- a/packages/openal-soft.cmake
+++ b/packages/openal-soft.cmake
@@ -18,7 +18,7 @@ ExternalProject_Add(openal-soft
         -DALSOFT_TESTS=OFF
         -DALSOFT_BACKEND_PIPEWIRE=OFF
     BUILD_COMMAND ${EXEC} ninja -C <BINARY_DIR>
-          COMMAND bash -c "echo 'Libs.private: -lole32' >> <BINARY_DIR>/openal.pc"
+          COMMAND bash -c "echo 'Libs.private: -lole32 -luuid -lshlwapi' >> <BINARY_DIR>/openal.pc"
     INSTALL_COMMAND ${EXEC} ninja -C <BINARY_DIR> install
     LOG_DOWNLOAD 1 LOG_UPDATE 1 LOG_CONFIGURE 1 LOG_BUILD 1 LOG_INSTALL 1
 )
-- 
2.43.0

