From 32b9e6b568bb21f47377101bb46bef77c28c9950 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Sun, 26 Apr 2026 00:11:47 +0800
Subject: [PATCH] mpv: change sha to 10 characters

---
 packages/mpv.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/mpv.cmake b/packages/mpv.cmake
index 51920c9..624037b 100644
--- a/packages/mpv.cmake
+++ b/packages/mpv.cmake
@@ -89,7 +89,7 @@ set(RENAME ${CMAKE_CURRENT_BINARY_DIR}/mpv-prefix/src/rename.sh)
 file(WRITE ${RENAME}
 "#!/bin/bash
 cd $1
-GIT=$(git rev-parse --short=7 HEAD)
+GIT=$(git rev-parse --short=10 HEAD)
 mv $2 $2-git-\${GIT}")
 
 ExternalProject_Add_Step(mpv copy-package-dir
-- 
2.52.0.windows.1

