From 3d453a97e32fba4a0385543e26597a68ee0a8ab3 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Sun, 25 May 2025 00:55:03 +0800
Subject: [PATCH] libarchive: set CMAKE_INSTALL_LIBDIR

---
 packages/libarchive.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/libarchive.cmake b/packages/libarchive.cmake
index ae1c0fa..bfd8126 100644
--- a/packages/libarchive.cmake
+++ b/packages/libarchive.cmake
@@ -37,6 +37,7 @@ ExternalProject_Add(libarchive
         -DENABLE_TEST=OFF
         -DWINDOWS_VERSION=WIN10
         -DCMAKE_POLICY_VERSION_MINIMUM=3.5
+        -DCMAKE_INSTALL_LIBDIR=lib
         "-DCMAKE_C_FLAGS='-lxml2 -lbz2 -llzo2 -lz -lbrotlienc -lbrotlidec -lbrotlicommon -lzstd -lws2_32 -lgdi32 -lcrypt32 -liconv -lbcrypt'"
     BUILD_COMMAND ${EXEC} ninja -C <BINARY_DIR>
     INSTALL_COMMAND ${EXEC} ninja -C <BINARY_DIR> install
-- 
2.48.1.windows.1

