From baf7e94472f476bd3e3dd3356159cab25d22b893 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Thu, 21 Sep 2023 11:30:10 +0800
Subject: [PATCH] libxml2: clamp to working commit

---
 packages/libxml2.cmake | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packages/libxml2.cmake b/packages/libxml2.cmake
index d54969f..a3c6b83 100644
--- a/packages/libxml2.cmake
+++ b/packages/libxml2.cmake
@@ -3,6 +3,9 @@ ExternalProject_Add(libxml2
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
     UPDATE_COMMAND ""
+    GIT_REMOTE_NAME origin
+    GIT_TAG master
+    GIT_RESET d7cfe356500906fad0f3b3d186e1abbd5489a23c
     CONFIGURE_COMMAND ${EXEC} <SOURCE_DIR>/configure
         --host=${TARGET_ARCH}
         --prefix=${MINGW_INSTALL_PREFIX}
-- 
2.42.0

