From 965dc8f732c40e03d49a0d70735000870e3c1a0e Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Sun, 6 Nov 2022 01:20:29 +0000
Subject: [PATCH] highway: clamp to working commit

---
 packages/highway.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/packages/highway.cmake b/packages/highway.cmake
index d2fe302..524aaf8 100644
--- a/packages/highway.cmake
+++ b/packages/highway.cmake
@@ -2,6 +2,8 @@ ExternalProject_Add(highway
     GIT_REPOSITORY https://github.com/google/highway.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_SHALLOW 1
+    GIT_REMOTE_NAME origin
+    GIT_TAG 4899d11ad66bc91659a70deb31d965acd86fe853
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} cmake -H<SOURCE_DIR> -B<BINARY_DIR>
         -DCMAKE_BUILD_TYPE=Release
-- 
2.38.1

