From 18726cdaa4439422bdb33780afbac9c2b67bc245 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Fri, 11 Apr 2025 21:19:00 +0800
Subject: [PATCH] x265: reset to working commit

---
 packages/x265.cmake | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packages/x265.cmake b/packages/x265.cmake
index 424f42f..2878a5c 100644
--- a/packages/x265.cmake
+++ b/packages/x265.cmake
@@ -11,6 +11,9 @@ ExternalProject_Add(x265
     GIT_REPOSITORY https://bitbucket.org/multicoreware/x265_git.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
+    GIT_REMOTE_NAME origin
+    GIT_TAG master
+    GIT_RESET b354c009a60bcd6d7fc04014e200a1ee9c45c167
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ""
     BUILD_COMMAND ""
-- 
2.48.1.windows.1

