From 329b174be09f38d6782a587d409388b5243bea43 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Wed, 3 Jan 2024 20:56:29 +0800
Subject: [PATCH] spirv-tools: reset to working commit

---
 packages/spirv-tools.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/spirv-tools.cmake b/packages/spirv-tools.cmake
index 8e1e44b..6a0a575 100644
--- a/packages/spirv-tools.cmake
+++ b/packages/spirv-tools.cmake
@@ -4,6 +4,7 @@ ExternalProject_Add(spirv-tools
     GIT_CLONE_FLAGS "--filter=tree:0"
     GIT_REMOTE_NAME origin
     GIT_TAG main
+    GIT_RESET f0cc85efdbbe3a46eae90e0f915dc1509836d0fc
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ""
     BUILD_COMMAND ""
-- 
2.42.0

