From f4be1c98ebca623d80f844654633e520be543dbc Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Wed, 18 Oct 2023 07:43:04 +0800
Subject: [PATCH] highway: clamp to working commit

---
 packages/highway.cmake | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packages/highway.cmake b/packages/highway.cmake
index 754519a..758cb1e 100644
--- a/packages/highway.cmake
+++ b/packages/highway.cmake
@@ -6,6 +6,9 @@ ExternalProject_Add(highway
     GIT_REPOSITORY https://github.com/google/highway.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
+    GIT_REMOTE_NAME origin
+    GIT_TAG master
+    GIT_RESET c4703916e078ce7dd264add83de5355aa5277c07
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} cmake -H<SOURCE_DIR> -B<BINARY_DIR>
         -DCMAKE_BUILD_TYPE=Release
-- 
2.41.0.windows.1

