From 2566346a530d0c8764af6237c472e2b96477f343 Mon Sep 17 00:00:00 2001
From: Andarwinux <Andarwinux@users.noreply.github.com>
Date: Thu, 1 Jan 1970 00:00:00 +0000
Subject: [PATCH] vulkan-header: disable cppm

---
 packages/vulkan-header.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/vulkan-header.cmake b/packages/vulkan-header.cmake
index bf18b3644..fcd9e829e 100644
--- a/packages/vulkan-header.cmake
+++ b/packages/vulkan-header.cmake
@@ -12,6 +12,7 @@ ExternalProject_Add(vulkan-header
         -DCMAKE_INSTALL_PREFIX=${MINGW_INSTALL_PREFIX}
         -DCMAKE_FIND_ROOT_PATH=${MINGW_INSTALL_PREFIX}
         -DBUILD_SHARED_LIBS=OFF
+        -DVULKAN_HEADERS_ENABLE_MODULE=OFF
     BUILD_COMMAND ""
     INSTALL_COMMAND ${EXEC} ninja -C <BINARY_DIR> install
     LOG_DOWNLOAD 1 LOG_UPDATE 1 LOG_CONFIGURE 1 LOG_INSTALL 1

     
