From 473c6d2aa76683ce9f94fc1f0d6a984089215444 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Sat, 8 Mar 2025 11:43:30 +0800
Subject: [PATCH] libarchive: reset to working commit

---
 packages/libarchive.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/libarchive.cmake b/packages/libarchive.cmake
index 4773fae..ae39923 100644
--- a/packages/libarchive.cmake
+++ b/packages/libarchive.cmake
@@ -11,6 +11,7 @@ ExternalProject_Add(libarchive
     GIT_REPOSITORY https://github.com/libarchive/libarchive.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
+    GIT_RESET 3512329ba9a06a7360601f195c5013c3161f5e70
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} CONF=1 cmake -H<SOURCE_DIR> -B<BINARY_DIR>
         -G Ninja
-- 
2.48.1

