From ba2df77fa6594822b0dc2b34ca43c45308c840ca Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Mon, 24 Jul 2023 12:46:24 +0800
Subject: [PATCH] mpv: dont copy nonexistent file

---
 packages/mpv.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/packages/mpv.cmake b/packages/mpv.cmake
index 27d5428..546365f 100644
--- a/packages/mpv.cmake
+++ b/packages/mpv.cmake
@@ -77,7 +77,6 @@ ExternalProject_Add_Step(mpv copy-binary
 
     COMMAND ${CMAKE_COMMAND} -E copy <BINARY_DIR>/libmpv-2.dll          ${CMAKE_CURRENT_BINARY_DIR}/mpv-dev/libmpv-2.dll
     COMMAND ${CMAKE_COMMAND} -E copy <BINARY_DIR>/libmpv.dll.a          ${CMAKE_CURRENT_BINARY_DIR}/mpv-dev/libmpv.dll.a
-    COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/libmpv/mpv.def        ${CMAKE_CURRENT_BINARY_DIR}/mpv-dev/mpv.def
     COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/libmpv/client.h       ${CMAKE_CURRENT_BINARY_DIR}/mpv-dev/include/mpv/client.h
     COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/libmpv/stream_cb.h    ${CMAKE_CURRENT_BINARY_DIR}/mpv-dev/include/mpv/stream_cb.h
     COMMAND ${CMAKE_COMMAND} -E copy <SOURCE_DIR>/libmpv/render.h       ${CMAKE_CURRENT_BINARY_DIR}/mpv-dev/include/mpv/render.h
-- 
2.41.0.windows.1

