From f834cb61157bd9cc796eea8b5d8473d262261965 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Thu, 8 May 2025 08:01:12 +0800
Subject: [PATCH] ffmpeg: remove --enable-postproc

---
 packages/ffmpeg.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/packages/ffmpeg.cmake b/packages/ffmpeg.cmake
index 49a0f6e..252c687 100644
--- a/packages/ffmpeg.cmake
+++ b/packages/ffmpeg.cmake
@@ -61,7 +61,6 @@ ExternalProject_Add(ffmpeg
         --enable-runtime-cpudetect
         --enable-gpl
         --enable-version3
-        --enable-postproc
         --enable-avisynth
         --enable-vapoursynth
         --enable-libass
-- 
2.48.1.windows.1

