From 3e0da0805164b84f54c171d2efa872e6d7b84c6e Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Wed, 18 Oct 2023 16:34:19 +0800
Subject: [PATCH] libmysofa: clamp to working commit

---
 packages/libmysofa.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/libmysofa.cmake b/packages/libmysofa.cmake
index 8f41b5a..dc2887d 100644
--- a/packages/libmysofa.cmake
+++ b/packages/libmysofa.cmake
@@ -4,6 +4,7 @@ ExternalProject_Add(libmysofa
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_REMOTE_NAME origin
     GIT_TAG main
+    GIT_RESET 202db87ef6138d1e4d2f663e981aee8163ee3a90
     GIT_CLONE_FLAGS "--sparse --filter=tree:0"
     GIT_CLONE_POST_COMMAND "sparse-checkout set --no-cone /* !tests"
     UPDATE_COMMAND ""
-- 
2.42.0
