From 197e0c19a1a3f1f54a66c1a6f8abf9644bf04df5 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Sun, 8 Mar 2026 20:54:43 +0800
Subject: [PATCH] openal-soft: fix build

---
 packages/openal-soft.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/openal-soft.cmake b/packages/openal-soft.cmake
index c32bdd6..8040b1a 100644
--- a/packages/openal-soft.cmake
+++ b/packages/openal-soft.cmake
@@ -13,6 +13,7 @@ ExternalProject_Add(openal-soft
         -DCMAKE_FIND_ROOT_PATH=${MINGW_INSTALL_PREFIX}
         -DBUILD_SHARED_LIBS=OFF
         -DLIBTYPE=STATIC
+        -DALSOFT_ENABLE_MODULES=OFF
         -DALSOFT_UTILS=OFF
         -DALSOFT_EXAMPLES=OFF
         -DALSOFT_TESTS=OFF
-- 
2.52.0.windows.1

