From c13f2554f2b273e7cbae698847a34193f0b79ecb Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Thu, 22 Jan 2026 20:16:47 +0800
Subject: [PATCH] svtav1: reset to working commit

---
 packages/svtav1.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/svtav1.cmake b/packages/svtav1.cmake
index c34545b..3dad6d1 100644
--- a/packages/svtav1.cmake
+++ b/packages/svtav1.cmake
@@ -2,6 +2,7 @@ ExternalProject_Add(svtav1
     GIT_REPOSITORY https://gitlab.com/AOMediaCodec/SVT-AV1.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
+    GIT_RESET f17a9e65b54668643547f75557ea0e1bc813a215
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} CONF=1 cmake -H<SOURCE_DIR> -B<BINARY_DIR>
         -G Ninja
-- 
2.52.0.windows.1

