From f27f5459f36af56e2b317f65bae0133ea30e746b Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Mon, 12 Jan 2026 21:35:51 +0800
Subject: [PATCH] openssl: reset to working commit

---
 packages/openssl.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/openssl.cmake b/packages/openssl.cmake
index 5bb526c..bcf1c95 100644
--- a/packages/openssl.cmake
+++ b/packages/openssl.cmake
@@ -7,6 +7,7 @@ ExternalProject_Add(openssl
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--sparse --filter=tree:0"
     GIT_CLONE_POST_COMMAND "sparse-checkout set --no-cone /* !test"
+    GIT_RESET 85645be82e469a815c033ed97889df27b7349648    
     GIT_SUBMODULES ""
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} sed -i [['/__int64 ossl_ssize_t/i #include <stdint.h>']] <SOURCE_DIR>/include/openssl/e_os2.h
-- 
2.52.0.windows.1

