From 84d6f39425a577e944a40cf08e3798f12f4ef319 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Wed, 13 May 2026 19:07:10 +0800
Subject: [PATCH] graphengine: update git repo

---
 packages/graphengine.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/graphengine.cmake b/packages/graphengine.cmake
index a2a53d0..0a701cc 100644
--- a/packages/graphengine.cmake
+++ b/packages/graphengine.cmake
@@ -1,5 +1,5 @@
 ExternalProject_Add(graphengine
-    GIT_REPOSITORY https://bitbucket.org/the-sekrit-twc/graphengine.git
+    GIT_REPOSITORY https://github.com/sekrit-twc/graphengine.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
     GIT_SUBMODULES ""
-- 
2.52.0.windows.1

