From 571a7ea17d5746cf1ab4bda85b05d4191b8fea15 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Sat, 14 Mar 2026 00:24:34 +0800
Subject: [PATCH] openssl: fix configure

---
 packages/openssl.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/packages/openssl.cmake b/packages/openssl.cmake
index d9b5f3c..523e93b 100644
--- a/packages/openssl.cmake
+++ b/packages/openssl.cmake
@@ -10,7 +10,8 @@ ExternalProject_Add(openssl
     GIT_SUBMODULES ""
     UPDATE_COMMAND ""
     PATCH_COMMAND ${EXEC} git am --3way ${CMAKE_CURRENT_SOURCE_DIR}/openssl-*.patch
-    CONFIGURE_COMMAND ${EXEC} CONF=1 <SOURCE_DIR>/Configure
+    CONFIGURE_COMMAND ${EXEC} mkdir -p apps/include
+    COMMAND ${EXEC} CONF=1 <SOURCE_DIR>/Configure
         --cross-compile-prefix=${TARGET_ARCH}-
         --prefix=${MINGW_INSTALL_PREFIX}
         --libdir=lib
-- 
2.52.0.windows.1

