From 683b0a20d69b4b777b583d8d63941649750f4af7 Mon Sep 17 00:00:00 2001
From: zhongfly <11155705+zhongfly@users.noreply.github.com>
Date: Sat, 28 Mar 2026 20:52:04 +0800
Subject: [PATCH] harfbuzz: disable utilities

---
 packages/harfbuzz.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/harfbuzz.cmake b/packages/harfbuzz.cmake
index 5248f25..410494e 100644
--- a/packages/harfbuzz.cmake
+++ b/packages/harfbuzz.cmake
@@ -18,6 +18,7 @@ ExternalProject_Add(harfbuzz
         -Dicu=disabled
         -Dglib=disabled
         -Dgobject=disabled
+        -Dutilities=disabled
         -Dtests=disabled
         -Ddocs=disabled
         -Dbenchmark=disabled
-- 
2.52.0.windows.1

