--- a/packages/ffmpeg.cmake
+++ b/packages/ffmpeg.cmake
@@ -46,7 +46,7 @@
         codec2
     )

-    GIT_REPOSITORY https://github.com/Andarwinux/FFmpeg.git
+    GIT_REPOSITORY https://github.com/librempeg/librempeg.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--depth=1 --sparse --filter=tree:0"
     GIT_PROGRESS TRUE
@@ -61,7 +61,6 @@
         --enable-gpl
         --enable-version3
         --enable-nonfree
-        --enable-postproc
         --enable-avisynth
         --enable-vapoursynth
         --enable-libass
