From f08638fd89f42f1579fce4e36cf67cd5b503d32d Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Sat, 16 Dec 2023 20:34:58 +0000
Subject: [PATCH] mpv: add patch

---
 packages/mpv.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/mpv.cmake b/packages/mpv.cmake
index 983719f..b534147 100644
--- a/packages/mpv.cmake
+++ b/packages/mpv.cmake
@@ -26,6 +26,7 @@ ExternalProject_Add(mpv
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
     UPDATE_COMMAND ""
+    PATCH_COMMAND ${EXEC} git am --3way ${CMAKE_CURRENT_SOURCE_DIR}/mpv-*.patch
     CONFIGURE_COMMAND ${EXEC} CONF=1 meson setup <BINARY_DIR> <SOURCE_DIR>
         --prefix=${MINGW_INSTALL_PREFIX}
         --libdir=${MINGW_INSTALL_PREFIX}/lib
